docs/cudf/source/user_guide/data-types.md

```diff
@@ -0,0 +1,156 @@
+# Supported Data Types
+
+cuDF supports many data types supported by NumPy and Pandas, including
+numeric, datetime, timedelta, categorical and string data types. We
+also provide special data types for working with decimals, list-like,
+and dictionary-like data.
+
+All data types in cuDF are [nullable](/user_guide/missing-data).
```

Comment:
Used a relative path instead

Reply:
A relative path is an improvement but not quite as good as `ref` syntax. If the page is moved or renamed, the `ref` will still pick it up as long as it's in the same Sphinx project. `ref`s with missing referents will throw warnings, and `ref`s are supported by non-HTML Sphinx outputs. An absolute/relative path won't be checked for correctness afaik.

Reply:
With markdown syntax `[nullable](mossing-data)`, I get:

```
cudf/docs/cudf/source/user_guide/data-types.md:8: WARNING: 'myst' reference target not found: mossing-data
```

The markdown syntax translates into Sphinx's [`any` role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-any), which seems pretty flexible in what it can do.

Reply:
Nice to know! If it's translated to the `any` role, that's awesome. I thought it would be interpreted as a link, rather than a reference.